src/us/kbase/workspace/database/Util.java

```diff
@@ -69,44 +106,4 @@ public static String checkString(
 	public static int codePoints(final String s) {
```

Comment:
I just noticed this is in the public api but doesn't have tests. Given it's so trivial and it's tested as part of the checkString tests I leave it up to you whether you want to bother

Reply:
I am happy to leave as-is.